.github/workflows/tests.yaml

```diff
@@ -413,10 +415,10 @@ jobs:
         user: ${{ secrets.ARTIFACTS_USER }}
         password: ${{ secrets.ARTIFACTS_PASSWORD }}
```

Comment:
this simply disables the `Upload logs to artifacts` step : should be on the job (`utapi-v2-tests`) itself